Tests/ContainerTest.php

```diff
@@ -22,13 +23,13 @@
 
 class ContainerTest extends TestCase
 {
-    public static function setUpBeforeClass(): void
+    public function testContainerWithDbalOnly(): void
     {
-        if (interface_exists(EntityManagerInterface::class)) {
-            return;
-        }
-
-        self::markTestSkipped('This test requires ORM');
+        $container = $this->createDbalOnlyTestContainer();
+        $this->assertInstanceOf(
+            LoggerChain::class,
+            $container->get('doctrine.dbal.logger.chain.default')
```

Comment:
I must have messed up somewhere because although the service definition has 2 loggers, that object has only one when I `dump` it.

Reply:
Oh but when I remove the first `else` branch in this PR, it works.

Reply:
See my comment on the broken code.

Reply:
I just pushed, does it look correct or do I still misunderstood? Sorry, I'm quite tired today :sweat_smile:

Reply:
This now looks good.

Reply:
[Pushed again](https://github.com/doctrine/DoctrineBundle/pull/1263#discussion_r524490742)